src/Gui/PreferencePackManager.cpp

```diff
@@ -232,6 +232,23 @@ void Gui::PreferencePackManager::importConfig(const std::string& packName,
     rescan();
 }
 
+// TODO(Shvedov): Is this suitable place for this method? It is more generic,
+// and maybe more suitable place at Application?
+std::vector<boost::filesystem::path> Gui::PreferencePackManager::modPaths() const
+{
+    auto userModPath = fs::path(App::Application::getUserAppDataDir()) / "Mod";
```

Comment:
What happens with userModPath? Shouldn't this be added to result?

Reply:
Oh! Of course it should! Strange, but this mistake did not brake previous behavior.

Reply:
To extend that question: there are about dozen other places in both C++ and Python code constructing path as `fs::path(App::Application::getUserAppDataDir()) / "Mod"`. Are those irrelevant to this PR intend?

Reply:
@3x380V  Actually, I ripgrepped all C++ code and except places of this change, there is one place left with "copy-to-clipboard" functionality of about dialog. This copy-to-clipboard functionality uses the folder names under the `"Mod"` as modules names, which is not suitable for  `--module-path` argument, because those names are not standardized. E.g. for nix the folders names are looks like `/nix/store/6n17hn0wj83ck9jg7nd8skaqf453q27d-source`.

Reply:
Done

Reply:
does this mean that copy to clipboard will not be giving accurate information anymore?

Reply:
It never did it before. You can check: try to run freecad with `--module-path` pointing to the correct module with Init.py outside the standard paths. The module will work, but will not appear in the clipboard. Probably, we should open an issue for that.

Reply:
@adrianinsaval, I added an issue #17843